src/js/components/toolbar/UndoRedo.tsx

```diff
@@ -0,0 +1,57 @@
+'use strict';
+// TODO Change from 'requires' to 'import'
+const historyActions = require('../../actions/historyActions');
+const undo = historyActions.undo;
+const redo = historyActions.redo;
+const assets = require('../../util/assets');
+const Icon = require('../Icon');
+
+import * as createReactClass from 'create-react-class';
+import * as propTypes from 'prop-types';
```

Comment:
this file appears not to use createReactClass or propTypes, so can probably remove these lines. (Also, we should generally be removing usages of createReactClass in favor of class .. extends React.Component.